Jenkinsfile_CNP

```diff
@@ -88,7 +88,7 @@ withPipeline(type, product, component) {
 
   // Vars needed for smoke / functional testing
   env.TEST_STORAGE_CONTAINER_NAME = 'bulkscan'
-  env.TEST_STORAGE_ACCOUNT_URL = 'https://reformscanaat.blob.core.windows.net'
+  env.TEST_STORAGE_ACCOUNT_URL = 'https://reformscanaatstaging.blob.core.windows.net'
```

Comment:
may we need 2 different values for pr and nonpr ?

Reply:
This is used for non PR functional tests. For PR's we use OSBA